src/Analytics.ts

```diff
@@ -144,6 +148,7 @@ export class Analytics {
     if (label) params.el = label
     if (value) params.ev = value
 
+    // eslint-disable-next-line no-console
```

Comment:
I think you can turn this into a warning instead, no need to disallow it I think